README.md

```diff
@@ -11,35 +11,29 @@
 
 ## What is PLANet?
 * PLANet is a time management library that automates the process of organizing your schedule.
-* PLANet takes your current schedule, your preferences and the different events and tasks that you want to add to that schedule and gives you the optimal schedule for your preferences.
+* PLANet takes your current schedule, preferences and different events and tasks that you want to add to that schedule and gives you the optimal schedule for your preferences.
 * PLANet lets you group together different tasks and time-slots according to category and importance so that your schedule suits your specific needs!
 * PLANet can split up different tasks so you can use your time more efficiently.
 
 ## How to use the library?
-The current upload format for this repository currently requires you to pull the repository into Android Studio and compile from there, though you are also welcome to take whichever files you want from this project and to add them to your code in any way you like :)
+The current upload format for this repository currently requires you to pull the repository into Android Studio and compile from there, though you are also welcome to take any file you want from the project and to use it in your code in any way you would like :)
 
 Steps:
-1. Clone git repo.
-2. Run the library and copy the created aar file from the generated folder.
-3. Paste the aar file into libs folder of your app.
-4. Add the following line to your module's gradle file:
-   In dependencies :
-   implementation(name: 'library-debug', ext: 'aar')
-   
-   In repositories :
-   flatDir {
-                             dirs 'libs'
-                         }
-5. Add the following permissions to your AndroidManifest:
-
-    <uses-permission android:name="android.permission.READ_CALENDAR" />
-    <uses-permission android:name="android.permission.WRITE_CALENDAR" />
-
-To add a new calendar just use our preCreated PlannerManager, which can be synced with
-google calendar automatically, or insert events manually, as specified during creation.
+1. Install Android Studio - Follow the instructions in https://developer.android.com/studio/install<br/>
```

Comment:
Please revert this section. 
This project is an android library. 
Android libraries are called from an app that wants to use them.
Because of that we wish to instruct that in order to use our library, the app must add the permissions to it's Manifest.

Reply:
done